### PR TITLE
fix: upgrade to swagger-ui-dist@3.12.1

### PR DESCRIPTION
**Pull request by raymondfeng**
- Upgrade swagger-ui
- Use https://buttons.github.io/ to display github star count

### CODE REVIEW

File: _includes/overview/top.html [hunk 1]
```diff
@@ -16,11 +16,14 @@ <h1 class="c-heading -h3">LoopBack</h1>
                 <a href="/resources">Resources</a></li>
               <li class="_nav-item {% if page.item == 'contributing' %}-active{% endif %}">
                 <a href="/contributing">Contributing</a></li>
+              <li class="_nav-item">
+                <!-- Place this tag where you want the button to render. -->
+                <a class="github-button" href="https://github.com/strongloop/loopback" data-show-count="true" aria-label="Star strongloop/loopback on GitHub">Star</a>
```
Comment (virkt25) [inline _includes/overview/top.html:21]: I'd change the label to `GitHub stars for strongloop/loopback`

File: style/index.html [hunk 2]
```diff
@@ -39,13 +41,16 @@ <h1 class="c-heading -h3">LoopBack</h1>
               <li class="_nav-item"><a href="/resources">Resources</a></li>
               <li class="_nav-item"><a href="/users">Who's using?</a></li>
               <li class="_nav-item"><a href="/contributing">Contributing?</a></li>
+              <li class="_nav-item">
+              <!-- Place this tag where you want the button to render. -->
+                <a class="github-button" href="https://github.com/strongloop/loopback" data-show-count="true" aria-label="Star strongloop/loopback on GitHub">Star</a>
```
Comment (virkt25) [inline style/index.html:48]: I'd change the label to GitHub stars for strongloop/loopback

File: style/index.html [hunk 3]
```diff
@@ -9,6 +9,8 @@
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="/css/prism.css">
     <link rel="stylesheet" href="/css/main.css">
+    <!-- Place this tag in your head or just before your close body tag. -->
```
Comment (virkt25) [inline style/index.html]: Comment line not needed

File: _layouts/default.html [hunk 4]
```diff
@@ -9,6 +9,8 @@
     </script>
     {% if page.datatable == true %}
     <link rel="stylesheet" type="text/css" href="//cdn.datatables.net/1.10.5/css/jquery.dataTables.css">
+    <!-- Place this tag in your head or just before your close body tag. -->
```
Comment (virkt25) [inline _layouts/default.html]: Comment line not needed

File: _includes/overview/head.html [hunk 5]
```diff
@@ -7,6 +7,8 @@
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="/css/overview/prism.css">
     <link rel="stylesheet" href="/css/overview/main.css">
+    <!-- Place this tag in your head or just before your close body tag. -->
```
Comment (virkt25) [inline _includes/overview/head.html]: Comment line not needed

File: _includes/head.html [hunk 6]
```diff
@@ -18,6 +18,9 @@
     <link rel="stylesheet" href="{{base}}/css/customstyles.css">
     <!-- link rel="stylesheet" href="/css/gcs.css" type="text/css"/ -->
 
+    <!-- Place this tag in your head or just before your close body tag. -->
```
Comment (virkt25) [inline _includes/head.html]: Comment line not needed

File: _includes/overview/top.html (hunk 1 above)
Comment (raymondfeng) [reply _includes/overview/top.html:21]: It doesn't really matter as users only see the icon.

**Review by virkt25 — APPROVED** [review]

**Review by b-admike — APPROVED** [review]
LGTM, the button looks great 👍

**Pull request merged**